src/interface.tsx

```diff
@@ -0,0 +1,23 @@
+export interface ITodoItem {
+  text: string;
+}
+
+export interface ITodoItemProps {
+  type: "todo" | "done";
+  todo: string;
+  idx: number;
+  deleteCurrentList: (idx: number) => void;
+  addToggleList: (text: string) => void;
+}
+
+export interface IState {
+  todoList: Array<ITodoItem>;
+  doneList: Array<ITodoItem>;
+  dispatch: React.Dispatch<IAction>;
```

Comment:
하나도 놓치지 않는 꼼꼼한 타이핑이 인상적이네요.

`Array<T>` 대신 `T[]`를 사용할 수 있답니다



```suggestion
  todoList: ITodoItem[];
  doneList: ITodoItem[];
  dispatch: React.Dispatch<IAction>;
```

Reply:
넵 감사합니다!